.github/workflows/code.yml

```
@@ -170,11 +170,45 @@ jobs:
       - name: Test cli works in runtime image
         run: docker run ${{ env.IMAGE_REPOSITORY }} --version
 
+  conda:
+    needs: [lint, dist, test]
```

```suggestion
    needs: [dist]
```
Container step needs the others as it might do a push on tag